lib/components/SideMenu/sort.ts

```diff
@@ -0,0 +1,23 @@
+export const endpointComparator = (a, b) => {
```

Comment:
Could you change this method to sort items **if only** `sortOrder` is specified?
Otherwise, this will introduce breaking changes as many ReDoc users rely on the current order (preserved from JSON)